packages/backend/src/open_payments/payment_pointer/service.test.ts

```diff
@@ -37,7 +37,6 @@ describe('Open Payments Payment Pointer Service', (): void => {
   let paymentPointerService: PaymentPointerService
   let accountingService: AccountingService
   let knex: Knex
-  let config: IAppConfig
```

Comment:
looks like we can remove line 48 as well (duplicate `config = await deps.use('config')` in `beforeAll`)